### DISCUSSION TITLE
Can't use server returned error message inside onError callback

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/livewire#1214] [3. Feature ideas]
### Description

Hello!

First of all thanks @calebporzio for the onError callback!

The reason of this ticket is that currently onError passes only the status code. But we have situations where we return the error message from backend, and we can have different messages for the same status code.

My suggestion is pass the entire response as argument to onError callback instead of passing only the status code. So we can customize the error message based on what the backend returned.

I can do a PR for this if it is ok. This seems to be simple to fix.

Thanks!

**Comment by lucasromanojf** [comment]
PR #4179

**Comment by lucasromanojf** [comment]
PR #4179 merged

**Reply by ariaieboy** [reply]
you can change your feature idea or bug report as answered when its been implemented or fixed. 
it can help us to have a clean disscussion section and it will be so easier to find ideas or bugs that really need to be fixed or implemented.

**Reply by joshhanley** [reply]
@ariaieboy you can't mark feature ideas as answered, only bug reports and help requests 🙂

**Reply by ariaieboy** [reply]
@joshhanley 🤔 i think we need more labels for the discussion section.
i am looking for good ideas or important bugs so i can contribute to the livewire but i am hitting alot of fixed or WIP disscussions.

if we could add more labels like issue section of the big projects it will help contributers to find good disscussions to start from there. 

like help-needed | good idea | confirmed bug etc

**Reply by ariaieboy** [reply]
i am not really familiar with github discussion but if it's have this feature we must use it more so we can filter discussions easier

**Reply by joshhanley** [reply]
@ariaieboy yeah that's something I've been wanting to improve for a while. I'll have a chat with Caleb and see how best to manage it so there is bugs and feature ideas available for anyone to look at the wants to contribute 🙂
